### DISCUSSION TITLE
Cryptomator not prompting for password

### DISCUSSION THREAD

**Discussion by PresidentJade** [iterate-ch/cyberduck#16686] [Q&A] [answered]
I just submitted a bug report in case it is a bug. But I thought Id ask here in case it was just me or if I may be misunderstanding something.

In Cyberduck > Preferences > Cryptomator I have both _Auto detect and open vault in browser_ and _Use Keychain_ unchecked.

And yet when I select the vault in the Filename directory and click on the **Unlock Vault** button on the Toolbar it opens the vault without prompting me for the password. Is this normal? Is your experience the same?

Thank you for letting me know.

**Comment by dkocher** [comment]
While the option _Use Keychain_ will disable saving passphrases in the password manager by default, lookup of passphrases previously saved is still enabled.

**Comment by tatze96** [comment, marked as answer]
If you want Cyberduck to prompt you for the password each time you access the respective vault again, remove the corresponding entries from _Keychain.app_ (macOS) or _Windows Credential Manager_.

**Comment by PresidentJade** [comment]
Thank you for the info and help.

I removed the corresponding entry from the Windows Credential Manager. Now when I open the vault (unlock vault button) the password prompt window opens non-stop multiple times over each other and I have to kill the Cyberduck processes in the Windows Task Manager to get it to stop.

Am I doing something wrong to cause this to happen?

**Reply by dkocher** [reply]
You see #16645 which will be fixed with 9.1.1 available shortly.
